src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -37,14 +37,20 @@
 from clusterfuzz._internal.system import shell
 
 MAX_TESTCASE_DIRECTORY_SIZE = 10 * 1024 * 1024  # in bytes.
+MAX_TESTCASES = 10000
 STORED_TESTCASES_LIST = []
 
 # pylint: disable=broad-exception-raised
 
 
 def unpack_crash_testcases(crash_testcases_directory):
   """Unpacks the old crash testcases in the provided directory."""
+  count = 0
   for testcase in ndb_utils.get_all_from_model(data_types.Testcase):
+    count += 1
+    if count >= MAX_TESTCASES:
+      logs.info(f'{MAX_TESTCASES} testcases reached.')
+      break
```

Comment:
What is the idea here?  This script will run several times and consume the old test cases slowly, or is this a hard cap untill some sort of corpus pruning task runs?

Reply:
Hard cap.

Reply:
There's never any pruning for here. This source has basically been polluted over the years and is unusable now.

Reply:
Maybe I will manually prune it, though I don't know how much it will move the needle.

Reply:
It'd be great if we could get the latest 500 though...